src/views/Pools/components/PoolCard/CardFooter/ExpandedFooter.tsx

```diff
@@ -75,12 +92,22 @@ const ExpandedFooter: React.FC<ExpandedFooterProps> = ({ pool, account }) => {
           </Flex>
         </Flex>
       )}
+      {isAutoVault && (
+        <Flex mb="2px" justifyContent="space-between" alignItems="center">
+          <Text fontSize="14px">{TranslateString(999, 'Performance Fee')}</Text>
+          <Flex alignItems="center">
+            <Text ml="4px" fontSize="14px">
+              {performanceFee / 100}%
+            </Text>
+          </Flex>
+        </Flex>
+      )}
       <Flex mb="2px" justifyContent="flex-end">
         <LinkExternal bold={false} fontSize="14px" href={earningToken.projectLink}>
           {TranslateString(412, 'View Project Site')}
         </LinkExternal>
       </Flex>
-      {poolContractAddress && (
+      {!isAutoVault && poolContractAddress && (
```

Comment:
No need to hide contract for auto I think.

Reply:
Yea nice, slipped the cake vault contract in there